tests/unit/migrations/namespace-form-fields.js

```diff
@@ -4,6 +4,39 @@ var sinon = require('sinon'),
     utils = require('../utils'),
     migration = require('../../../migrations/namespace-form-fields');
 
+// Arguments : docArrayForFirstBatch, docArrayForSecondBatch, ...
+var makeStubs = function() {
+  var getView = sinon.stub(db.medic, 'view');
+  if (arguments.length === 0) {
+    getView.callsArgWith(3, null,
+      {
+        total_rows: 0,
+        rows: []
+      });
+  } else {
+    var viewBatches = [];
+    var totalRows = 0;
+    for (var i = 0; i < arguments.length; i++) {
```

Comment:
I knooooooooow
But arguments isn't actually an array, it's this weird object. So the reduce breaks. Very disappointing.